src/AppBundle/Controller/RepositoryController.php

```diff
@@ -40,13 +41,49 @@ public function indexAction(Request $request, Repository $repository)
             }
         }
 
+        $user = $this->getUser();
+        $repositoryStar = $this->get('doctrine')->getRepository('AppBundle:RepositoryStar')->findOneBy([
```

Comment:
IMO, if we create a method `isStarredByUser($repo, $user)`, it should be in RepositoryStarRepository. Like `getLatestPublic`, `getMostStared` or `getMostPulled` of RepositoryRepository.
With this solution, we can choose between :
 * 2 methods `isStarredByUser($repo, $user)` and `removeRepositoryUser($repo, $user)`
 * 1 method `findRepositoryStarByUser($repo, $user)` and work with the result (entity or null)